File: Python/tests/test_bart_withintrial.py
import pytest

from hbayesdm.models import bart_withintrial


def test_bart_withintrial():
    _ = bart_withintrial(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_bart_withintrial.R
context("Test bart_withintrial")
library(hBayesDM)

test_that("Test bart_withintrial", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_withintrial(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
